sonic-net/sonic-utilities
# [NTP] Configure NTP according to extended configuration

hld [#1219](https://github.com/sonic-net/SONiC/pull/1296)
closes [#1171](https://github.com/sonic-net/SONiC/issues/1254)

#### What I did
Fix NTP CLI according to new extended configuration abilities

#### How I did it
Additionally configure `association_type` and `resolve_as` when adding new NTP servers.

#### How to verify it
* Add a new server
* Show NTP sync `show ntp`. It should show you are sync with new server
* Delete NTP server
* Show NTP sync `show ntp`. It should show you are unsync

#### Previous command output (if the output of a command-line utility has changed)

#### New command output (if the output of a command-line utility has changed)
Not changed
